plugins/jobs/src/js/components/JobsOverviewTable.js

```diff
@@ -258,16 +288,16 @@ class JobsTable extends React.Component {
         className="table table-flush table-borderless-outer table-borderless-inner-columns table-hover flush-bottom"
         colGroup={this.getColGroup()}
         columns={this.getColumns()}
-        data={this.getData()}
+        data={this.getData(this.props.data)}
         itemHeight={TableUtil.getRowHeight()}
         sortBy={{ prop: "name", order: "asc" }}
       />
     );
   }
 }
 
-JobsTable.propTypes = {
-  jobs: PropTypes.array.isRequired
+JobsOverviewTable.propTypes = {
+  data: PropTypes.object.isRequired // JobConnection
```

Comment:
🤔 Would it make sense to use `PropTypes.shape` here to detect problems with our type assumptions?

Reply:
Offline discussed: won't do